modules/platforms/dotnet/Apache.Ignite.Core.Tests/Dataload/DataStreamerTest.cs

```diff
@@ -502,6 +502,27 @@ public void TestStreamTransformer()
             TestStreamReceiver(new StreamTransformer<int, int, int, int>(new EntryProcessorBinarizable()));
         }
 
+        [Test]
```

Comment:
Dont we need a separate test for local DataStreamer (with pointer allocation)?

Reply:
This is covered by existing tests.